app/models/course.rb

```diff
@@ -3,7 +3,9 @@ class Course < ActiveRecord::Base
 
   scope :search, ->(q) { where("concat(lower(courses.department), courses.course_number) like ?
                                OR lower(courses.title) like ?",
-                               "%#{q.to_s.downcase.gsub(/\s+/,'')}%", "%#{q.to_s.downcase}%") }
+                               "%#{q.to_s.downcase.gsub(/\s+/,'')}%", "%#{q.to_s.downcase}%")
+                                .order(%{concat(lower(courses.department), courses.course_number)})
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 30) spaces for indenting an expression spanning multiple lines.